src/test/java/org/junitpioneer/jupiter/AbstractEntryBasedExtensionTests.java

```diff
@@ -18,7 +18,7 @@
 import org.junitpioneer.testkit.PioneerTestKit;
 
 @DisplayName("Abstract entry-based extension")
-class AbstractEntryBasedExtensionTest {
+class AbstractEntryBasedExtensionTests {
```

Comment:
Good catch!

There's still an issue (our fault again): the test sometimes fails due to concurrent execution. To prevent this, please add the following to the class:

```java
@WritesEnvironmentVariable
@WritesSystemProperty
```